working-groups/WORKING-GROUPS.md

```diff
@@ -225,6 +225,10 @@ interest include: load balancing, routing, DNS configuration and TLS support.
 | <img width="30px" src="https://github.com/nak3.png">        | Kenjiro Nakayama | Red Hat   | [nak3](https://github.com/nak3)               |
 | <img width="30px" src="https://github.com/ZhiminXiang.png"> | Zhimin Xiang     | Google    | [ZhiminXiang](https://github.com/ZhiminXiang) |
 
+| &nbsp;                                                  | Emeritus Leads             | Profile                               | Duration  |
+| --------------------------------------------------------| -------------------------- | --------------------------------------| --------- |
+| <img width="30px" src="https://github.com/tcnghia.png"> | Nghia Tran (Technical)     | [tcnghia](https://github.com/tcnghia) | 2018-2021 |
```

Comment:
do we need the (Technical) here? I don't think networking ever had a split between Technical and Execution lead like eventing did (but I could be wrong - just checking!)

Reply:
For the client emeritus list we also don't use the distinction between technical/non-technical. I think its only needed for WGs that leverage this concept.

Reply:
Yes, agreed 👍

Reply:
addressed the comment.